src/argilla/server/daos/backend/mappings/helpers.py

```diff
@@ -14,14 +14,13 @@
 
 from typing import Any, Dict, List
 
+from argilla.server.daos.backend.mappings.stopwords import english
 from argilla.server.settings import settings
 
 EXTENDED_ANALYZER_REF = "extended_analyzer"
 
 MULTILINGUAL_STOP_ANALYZER_REF = "multilingual_stop_analyzer"
 
-DEFAULT_SUPPORTED_LANGUAGES = ["es", "en", "fr", "de"]  # TODO: env var configuration
-
 
 class mappings:
```

Comment:
Class name should be upper case?

Reply:
Yes, we will tackle it in another place. This class contains only static methods, so maybe can be converted into a module itself.

Reply:
Got it